CMakeLists.txt

```diff
@@ -35,23 +35,21 @@ add_definitions(-DNOMINATIM_VERSION="${NOMINATIM_VERSION}")
 
 set(BUILD_TESTS off CACHE BOOL "Build test suite" FORCE)
```

Comment:
The situation with BUILD_TESTS is a bit difficult. The variable here is forced to off to avoid building the tests for osm2pgsql. So it actually should be 'off' in a non-configurable way. If you want the Nominatim tests to be configurable they should get a different variable.